streaming/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@mastodon/streaming",
+  "license": "AGPL-3.0-or-later",
+  "description": "Mastodon's Streaming Server",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mastodon/mastodon.git"
+  },
+  "scripts": {
+    "start": "node ./index.js"
+  },
+  "dependencies": {
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "is-docker": "2.2.1",
+    "jsdom": "^22.0.0",
+    "npmlog": "^7.0.1",
+    "pg": "^8.5.0",
+    "pg-connection-string": "^2.5.0",
+    "redis": "^4.6.5",
+    "uuid": "^9.0.0",
+    "ws": "^8.12.1"
+  },
+  "optionalDependencies": {
+    "bufferutil": "^4.0.7",
+    "utf-8-validate": "^6.0.3"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.17",
+    "@types/npmlog": "^4.1.4",
+    "@types/pg": "^8.6.6",
+    "@types/uuid": "^9.0.1",
+    "prettier": "^2.8.8",
+    "typescript": "^5.0.4"
```

Comment:
These may not be needed, but I'm playing it safe by including them here, such that we can move the streaming server towards typescript in the future (I've seen some issues where missing these in mono-repos can result in them not being found by tooling and editors, despite the fact that they should be being inherited from the root package.json)